### PR TITLE
Adjust material requirement, add deflated mass and crew requirement.

### PR DESCRIPTION
**Pull request by PulsarAero**

### CODE REVIEW

File: GameData/StationPartsExpansionRedux/Patches/SSPXR-USILS.cfg [hunk 1]
```diff
@@ -4,20 +4,32 @@
...
 	@MODULE[ModuleDeployableCentrifuge]
 	{
 		DeployResource = MaterialKits
-		DeployResourceAmount = #$../mass$
-		@DeployResourceAmount *= 1000
+		DeployResourceAmount = #$../DeployedMass$
+		@DeployResourceAmount *= 600
+		DeployedMassModifier = #$../DeployedMass$
+		@DeployedMassModifier *= 0.6
+		CrewToDeploy = 1
```
Comment (ChrisAdderley) [inline GameData/StationPartsExpansionRedux/Patches/SSPXR-USILS.cfg]: You should use @ or % here, the base mod defines a requirement already.

File: GameData/StationPartsExpansionRedux/Patches/SSPXR-USILS.cfg [hunk 2]
```diff
@@ -4,20 +4,32 @@
...
 	@MODULE[ModuleDeployableHabitat]
 	{
 		DeployResource = MaterialKits
-		DeployResourceAmount = #$../mass$
-		@DeployResourceAmount *= 1000
+		DeployResourceAmount = #$../DeployedMass$
+		@DeployResourceAmount *= 600
+		DeployedMassModifier = #$../DeployedMass$
+		@DeployedMassModifier *= 0.6
+		CrewToDeploy = 1
```
Comment (ChrisAdderley) [inline GameData/StationPartsExpansionRedux/Patches/SSPXR-USILS.cfg]: Ditto.

**Comment by PulsarAero** [comment]
I edited those syntaxes a bit. I believe DeployResource and DeployResourceAmount are already defined as well?

**Comment by ChrisAdderley** [comment]
They won't be, but using % never hurts :).

**Pull request merged**